episodes/continuous-integration.md

```diff
@@ -105,6 +105,16 @@ Running applications in containers tends to enforce decoupling from external dep
 Building and distributing containers is generally done using [docker](https://www.docker.com) but there are others such as [podman](https://podman.io/)
 You can find more information about containers [here](https://book.the-turing-way.org/reproducible-research/renv/renv-containers.html).
 
+Publishing your application as a docker image is advised if the application has a lot of dependencies or if the build process is very complicated.
+If the application is simple with only few dependencies, then creating a docker image probably creates too much overhead.
+Docker is not suitable for publishing a module or library.
+Include a link to a tutorial on creating docker images.
```

Comment:
```suggestion
Here is a tutorial on 
[how to build docker images](https://book.the-turing-way.org/reproducible-research/renv/renv-containers.html#building-images-and-dockerignore-files)
```

Reply:
Ooops, that reminder slipped under the radar. :flushed:

Reply:
For the rest, I think it looks fine.